source/MenuAnimationPanel.cpp

```diff
@@ -28,6 +28,7 @@ MenuAnimationPanel::MenuAnimationPanel()
 	SetTrapAllEvents(false);
 
 	Audio::Play(Audio::Get("landing"));
+	Audio::PlayMusic("music/MAIN_SCREEN");
```

Comment:
Why is it all caps? Is that the new convention?

Reply:
no